### DISCUSSION TITLE
SpeechtoText - get back the raw audio recorded

### DISCUSSION THREAD

**Discussion by whann0205** [CommunityToolkit/Maui#2138] [New Feature Discussions]
After i have implemented the SpeechtoText Component the
customer told me if its possible to hear what was spoken
to review the audio. Is that possible now, if not can it be done
by myself or is it a new feature?

**Comment by Darlanio** [comment]
Getting the audio as a stream (WAV?) rather than having it spoken would solve this AND make it possible to alter the sound (eg. making it slower/faster): https://github.com/dotnet/maui/issues/14851

**Comment by whann0205** [comment]
Continued on 
https://github.com/CommunityToolkit/Maui/discussions/2683
